CHANGELOG.md

```diff
@@ -45,8 +48,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
-- `encode_global_metrics` (replaced by `prometheus_exporter::encode_to_string`)
-- `global_metrics_exporter` (replaced by `prometheus_exporter::init`)
+- `encode_global_metrics` was removed and replaced by `prometheus_exporter::encode_to_string`
+- `global_metrics_exporter` was removed and replaced by `prometheus_exporter::init`
+- `backends::prometheus_client::REGISTRY` was removed. The `Registry` can now be accessed as
+  a property on the `AutometricsSettings` struct
```

Comment:
From the point of view of the users, the registry is only accessible if they chose the `prometheus-client` feature, I don't know if we should specify the feature in the CHANGELOG, or make the registry in `AutometricsSettings::prometheus_registry` `pub` instead of `pub(crate)`, but it feels a little inconsistent

Reply:
I added a getter method on the settings for the `prometheus::Registry` as well